### ISSUE TITLE
What does hosting look like?

### ISSUE THREAD

**Issue by pbugnion** [voila-gallery/gallery#7]
For now, I have registered the domain `voila-gallery.org`. This currently points to an EC2 instance.

A more permanent solution to this would be good.

**Comment by pbugnion** [comment]
@yuvipanda Happy to create a sub-account that is linked to my AWS billing, but where you can have an admin role if that'd be useful.

**Comment by yuvipanda** [comment]
@SylvainCorlay mentioned OVH might be interested in hosting this free of cost.

Thanks for setting this up, @pbugnion! Giving me access sounds like a good idea.

**Comment by jtpio** [comment]
@pbugnion we got access to OVH and can now host the voila gallery there.

Just deployed the latest master (https://github.com/voila-gallery/gallery/commit/ed7c9a67afc3a71278b9596c68f147ed3fdebf3b) on a single `b2-30` instance (30GB RAM, 8 vCores, 200GB SSD): http://51.77.206.191/services/gallery/

How about pointing `voila-gallery.org` to `51.77.206.191`?

There are also discussions on having `voila.ovh.com` or even `voila.ovh`.

**Comment by yuvipanda** [comment]
I like voila-gallery.org best :D

**Comment by pbugnion** [comment]
I have switched `voila-gallery.org` to `51.77.206.191`:

<img width="710" alt="Screenshot 2019-06-16 at 08 18 35" src="https://user-images.githubusercontent.com/1392879/59560821-c37a6b80-900f-11e9-96c7-8a38690b7851.png">

I personally like `voila-gallery.org` best. If OVH wants credit for hosting it (which is completely reasonable), we could consider creating a banner somewhere that says 'hosted by OVH'?

I think we can close this? I've created two new issues:
- #22 deals with crediting OVH for hosting
- #21 deals with managing ownership of the voila-gallery.org domain
